### DISCUSSION TITLE
Add DMA support to SD Host

### DISCUSSION THREAD

**Discussion by endinferno** [rsta2/circle#315] [Ideas]
I just find that the part of SD Host code is the same as Linux bcm2835-sdhost.
Linux bcm2835 have DMA support, so I am trying to support DMA in circle SD Host.
Can I use CDMAChannel in circle to implement what I am talking about and will it be a easy thing?

**Comment by rsta2** [comment]
I wouldn't call it easy, but I'm not aware of a reason, why porting the DMA support should not be possible. Yes, the class `CDMAChannel` has to be used for implementing DMA transfers with the platform DMA controller.
